plugins/spf.js

```diff
@@ -66,13 +71,18 @@ exports.load_config = function () {
     if (!plugin.cfg.relay) {
         plugin.cfg.relay = { context: 'sender' };  // default/legacy
     }
-}
+
+    plugin.cfg.lookup_timeout = plugin.cfg.main.lookup_timeout || 30;
+};
 
 exports.hook_helo = exports.hook_ehlo = function (next, connection, helo) {
     const plugin = this;
 
     // Bypass private IPs
-    if (connection.remote.is_private) { return next(); }
+    if (net_utils.is_private_ip(connection.remote_ip)) {
```

Comment:
Why switch this to using a more expensive function call?  The precomputed property is always set and accessing it will always be faster than a function call.

Reply:
Take it easy, the patch is from 2016. I'll adjust the stuff, don't worry.